src/view/position.js

```diff
@@ -176,6 +176,25 @@ export default class Position {
 	}
 
 	/**
+	 * Returns a {@link module:engine/view/node~Node} which is a common ancestor for both positions.
+	 *
+	 * @param {module:engine/view/position~Position} position
+	 * @returns {module:engine/view/node~Node|null}
+	 */
+	getCommonAncestor( position ) {
+		const ancestorsA = this.getAncestors();
```

Comment:
`getAncestors()` accept `options.includeNode` so will this work in such a case: `<p>x{y}z</p>`? For such positions the text node should be returned. And for such `<p>[x]</p>` the element.